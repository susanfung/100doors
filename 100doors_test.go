package main

import (
	"testing"
	approvals "github.com/approvals/go-approval-tests"
)

func TestInitialState(t *testing.T) {
	approvals.VerifyArray(t, Answer(0))
}

func TestFirstPass(t *testing.T) {
	approvals.VerifyArray(t, Answer(1))
}

func TestSecondPass(t *testing.T) {
	approvals.VerifyArray(t, Answer(2))
}