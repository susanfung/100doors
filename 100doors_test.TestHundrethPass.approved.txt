[0]=opened
[1]=closed
[2]=closed
[3]=opened
[4]=closed
[5]=closed
[6]=closed
[7]=closed
[8]=opened
[9]=closed
[10]=closed
[11]=closed
[12]=closed
[13]=closed
[14]=closed
[15]=opened
[16]=closed
[17]=closed
[18]=closed
[19]=closed
[20]=closed
[21]=closed
[22]=closed
[23]=closed
[24]=opened
[25]=closed
[26]=closed
[27]=closed
[28]=closed
[29]=closed
[30]=closed
[31]=closed
[32]=closed
[33]=closed
[34]=closed
[35]=opened
[36]=closed
[37]=closed
[38]=closed
[39]=closed
[40]=closed
[41]=closed
[42]=closed
[43]=closed
[44]=closed
[45]=closed
[46]=closed
[47]=closed
[48]=opened
[49]=closed
[50]=closed
[51]=closed
[52]=closed
[53]=closed
[54]=closed
[55]=closed
[56]=closed
[57]=closed
[58]=closed
[59]=closed
[60]=closed
[61]=closed
[62]=closed
[63]=opened
[64]=closed
[65]=closed
[66]=closed
[67]=closed
[68]=closed
[69]=closed
[70]=closed
[71]=closed
[72]=closed
[73]=closed
[74]=closed
[75]=closed
[76]=closed
[77]=closed
[78]=closed
[79]=closed
[80]=opened
[81]=closed
[82]=closed
[83]=closed
[84]=closed
[85]=closed
[86]=closed
[87]=closed
[88]=closed
[89]=closed
[90]=closed
[91]=closed
[92]=closed
[93]=closed
[94]=closed
[95]=closed
[96]=closed
[97]=closed
[98]=closed
[99]=opened